ddr/tools/getmacros

```diff
@@ -30,6 +30,7 @@ default_regex="@ddr_namespace: *default"
 map_to_type_regex="@ddr_namespace: *map_to_type="
 flag_define_regex="^ *# *define +(${id}) *($|//|/\*)"
 flag_undef_regex="^ *# *undef +(${id}) *($|//|/\*)"
+flag_undef_regex_comment="^/\* #undef (${id}) \*/"
```

Comment:
This should be `"^/\/\* #undef (${id}) \*\//"` (note the leading and trailing (escaped) `/`).

Reply:
That matches a literal `//*` at the start of a line, and a `*//` at the end of our match.

This is the result of my test(on AIX), expanding out $id:
```
-bash-4.4$ pattern="^/\/\* #undef ([A-Za-z_][A-Za-z0-9_]*) \*\//"
-bash-4.4$ echo "$pattern"
^/\/\* #undef ([A-Za-z_][A-Za-z0-9_]*) \*\//
-bash-4.4$ string="/* #undef FLAG */"
-bash-4.4$ echo "$string"
/* #undef FLAG */
-bash-4.4$ if [[ $string =~ $pattern ]]; then echo "found"; else echo "not found"; fi;
not found
-bash-4.4$ string="//* #undef FLAG *//"
-bash-4.4$ if [[ $string =~ $pattern ]]; then echo "found"; else echo "not found"; fi;
found
-bash-4.4$ string="/* #undef FLAG */"
-bash-4.4$ pattern="^/\* #undef ([A-Za-z_][A-Za-z0-9_]*) \*/"
-bash-4.4$ if [[ $string =~ $pattern ]]; then echo "found"; else echo "not found"; fi;
found
```

Reply:
My apologies, I thought that pattern was for `sed`, not the shell: the current `flag_undef_regex_comment` is good.